### ISSUE TITLE
Lazy slide rendering ?

### ISSUE THREAD

**Issue by ju1ius** [souporserious/react-view-pager#18]
Hi,

The 0.3 version had a very nice feature: only the current & incoming/outgoing slides were rendered.
This made it well suited for easily displaying long lists of media-heavy slides (e.g fullscreen images, videos...). Unfortunately, it seems this behaviour is gone in 0.4...

Can this be accomplished with the 0.4 version ?
Is there a way to detect if a slide is currently visible ?
Do you plan on re-introducing this behaviour in future releases ?

**Comment by souporserious** [comment]
Yes I plan to bring this back. The challenge right now is trying to render a slide instantly as it comes in from the left or the right while keeping the physics motion. Accomplishing this would allow for multiple things like lazy slide rendering and some other cool things. I'm open to any PR's if you want to take a stab at it. I'm currently caught up for the next few weeks. If you want to see what the old version looked like you can see it at this branch here: https://github.com/souporserious/react-motion-slider/tree/instant-slider

**Comment by undefinedExpert** [comment]
Hi @souporserious i could help you out with this, do you've any idea how it might work? Also i wish ask about current status of this issue, are you working on it?

**Comment by souporserious** [comment]
@lanskey I have not had a chance to work on this yet. I've been meaning to do a rewrite so this library works more like [Flickity](http://flickity.metafizzy.co/) then work on adding lazy slide rendering back in after.

**Comment by souporserious** [comment]
@ju1ius sorry it's been so long on this issue, now that I have it in a better place I will start trying to figure out lazy rendering again.
